### ISSUE TITLE
Failing l10n test

### ISSUE THREAD

**Issue by vbudhram** [mozilla/fxa-auth-server#2067]
Noticed that this [test](https://circleci.com/gh/mozilla/fxa-auth-server/1195?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) is failing on master locally and https://github.com/mozilla/fxa-auth-server/pull/2066.

```
      AssertionError: not en-US from assert(emailData.text.indexOf('Activate now') === -1, 'not en-US')
      + expected - actual

      -false
      +true
```

Looking through the localization files, they appear to been [commented out](https://github.com/mozilla/fxa-content-server-l10n/blob/master/locale/pt_BR/LC_MESSAGES/server.po#L326)? There is probably something I am missing on how this was working previously, and whey the recent PRs did no fail this test.

**Comment by rfk** [comment]
> Looking through the localization files, they appear to been commented out?

Indeed, this commit from 2 days ago appears to have removed the translation for this string:

  https://github.com/mozilla/fxa-content-server-l10n/commit/eb3c539542a30581c588f1b99c435759e4ef0b05

Could be a bug in the string extraction process, maybe triggered by recent version/dependency upgrades?  @vladikoff thoughts?

**Comment by vladikoff** [comment]
Yeah, l10n options have changed with the module update. Thanks, should be fixed now
